021-quarkus-panache-multiple-pus/src/main/java/io/quarkus/qe/multiplepus/NotFoundExceptionMapper.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkus.qe.multiplepus;
+
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+// workaround for Quarkus providing its own NotFoundExceptionMapper
+// which is more specific than our ApplicationExceptionMapper
```

Comment:
Can you add a link to this issue? Maybe https://github.com/quarkusio/quarkus/issues/15272 ?

Reply:
Fixed.